data/test_domains/test_domain_from_directory2/domain_2.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.0"
```

Comment:
`domain_2.yml` can also be changed so that it describes the domain contents? `ban_weapons.yml` can be a good one. Though I must say seeing weapons in a file cringes me a little, but it might just be me (I hate anything to do with war)?